### PR TITLE
Add Xperia 10 II

### PR DESCRIPTION
**Pull request by sledges**
- `seine: fix audio` needs ~~https://github.com/mer-hybris/pulseaudio-modules-droid/pull/98~~
- `rpm: vendor is now bundled with system` needs ~https://github.com/mer-hybris/droid-system-device/pull/16 and its submodule update~.
- `sparse: move non device-specific files to submodule` needs ~https://github.com/mer-hybris/droid-hal-configs/pull/199~

### CODE REVIEW

File: kickstart/pack/xqau52/hybris [hunk 1]
```diff
@@ -100,36 +93,7 @@ dd if=home.img bs=$BLOCKSIZE count=$HOMEBLOCKS of=/dev/sailfish/home
 
 /usr/sbin/vgchange -a n sailfish
 
-pigz -7 root.img && md5sum -b root.img.gz > root.img.gz.md5&
```
Comment (mkosola) [inline kickstart/pack/xqau52/hybris:103]: We don't have factory image?

File: patterns/patterns-sailfish-device-adaptation-pdx201.inc [hunk 2]
```diff
@@ -57,18 +63,15 @@ Requires: usb-moded
 Requires: rfkill
 
 # enable device lock and allow to select untrusted software
-Requires: sailfish-devicelock-fpd
```
Comment (mkosola) [inline patterns/patterns-sailfish-device-adaptation-pdx201.inc:60]: We don't have fingerprint reader?

File: patterns/patterns-sailfish-device-configuration-xqau51.inc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Requires: droid-config-xqau51-preinit-plugin
+Requires: droid-config-xqau51-flashing
+Requires: droid-config-xqau51-bluez5
+Requires: droid-hal-version-xqau51
+
+Requires: patterns-sailfish-cellular-apps
+
+Requires: sailfish-content-graphics-z%{icon_res}
+
+Requires: jolla-settings-networking-multisim
```
Comment (mkosola) [inline patterns/patterns-sailfish-device-configuration-xqau51.inc:18]: This should be single sim?

File: kickstart/pack/xqau52/hybris (hunk 1 above)
Comment (sledges) [reply kickstart/pack/xqau52/hybris:103]: No, we don't have partition to flash it to. Factory images on Sailfish X should be used available flashing images from the website to reflash device.

File: patterns/patterns-sailfish-device-adaptation-pdx201.inc (hunk 2 above)
Comment (sledges) [reply patterns/patterns-sailfish-device-adaptation-pdx201.inc:60]: We have now, will fix.

File: kickstart/pack/xqau52/hybris (hunk 1 above)
Comment (mkosola) [reply kickstart/pack/xqau52/hybris:103]: OK

**Review by mkosola — APPROVED** [review]

**Pull request merged**